containerd-shim-spin/src/engine.rs

```diff
@@ -272,6 +320,16 @@ impl SpinEngine {
         let executor = builder.build(locked_url, runtime_config, init_data).await?;
         Ok(executor)
     }
+
+    // Returns Some(WasmLayer) if the layer contains wasm, otherwise None
+    fn is_wasm_content(layer: &WasmLayer) -> Option<WasmLayer> {
```

Comment:
nit: I think this function should move to upstream containerd-shim-wasm crate. we can do it later though.

Reply:
I don't think we can currently since the `OCI_LAYER_MEDIA_TYPE_WASM` is custom layer type for Spin.